xpresso/openapi/_builder.py

```diff
@@ -45,7 +45,7 @@
         "detail": {
             "title": "Detail",
             "type": "array",
-            "items": {"$ref": REF_PREFIX + "ValidationError"},
+            "items": {"$ref": f"{REF_PREFIX}ValidationError"},
```

Comment:
👍, and thanks for looking though this module, I think it's very messy and probably needs several refactors to tidy up. At least it's private so we can do whatever we want in the future 😄